### PR TITLE
Safelist Content-Length header

### PR DESCRIPTION
**Pull request by benjamingr**
Fixes: https://github.com/whatwg/fetch/issues/622

***
[Preview](https://s3.amazonaws.com/pr-preview/benjamingr/fetch/patch-1.html) | [Diff](https://s3.amazonaws.com/pr-preview/whatwg/fetch/8ab040a...benjamingr:3908d31.html)

### CODE REVIEW

File: fetch.bs [hunk 1]
```diff
@@ -472,7 +473,6 @@ is a <a>byte-case-insensitive</a> match for one of
  <li>`<a http-header><code>Access-Control-Request-Headers</code></a>`
  <li>`<a http-header><code>Access-Control-Request-Method</code></a>`
  <li>`<code>Connection</code>`
- <li>`<code>Content-Length</code>`
```
Comment (annevk) [inline fetch.bs]: Please don't remove this, it's still a forbidden header name.

**Comment by annevk** [comment]
For tests I suspect there are resources that need changes, `fetch/api/cors/cors-filtering.js` for sure. Not entirely sure what else. Fixing that, adding some tests, and noting in implementation bugs that there might be more fallout we haven't spotted is sufficient I think.

**Comment by annevk** [comment]
Yeah, for normative changes there also needs to be a PR against web-platform-tests ensuring the change is tested (and any existing tests that need to be modified are modified) and browser bugs need to be filed pointing to both PRs ensuring everyone is notified.

Let me know how much of that you're willing to take on. I can help.

**Comment by annevk** [comment]
At a high level this is documented at https://whatwg.org/working-mode#changes by the way, but perhaps we should also have a more concrete day-to-day guide.

**Comment by benjamingr** [comment]
> Let me know how much of that you're willing to take on. I can help.

I'm looking at this as an opportunity to learn the process better - if this gets annoying feel free to go ahead and make changes. I'll try to follow up with the tests tomorrow morning.

**Comment by annevk** [comment]
@benjamingr did you get around to making tests?

**Comment by benjamingr** [comment]
@annevk sorry, not yet :(

**Comment by shacharz** [comment]
Chrome - https://bugs.chromium.org/p/chromium/issues/detail?id=841308
Firefox - https://bugzilla.mozilla.org/show_bug.cgi?id=1460299
Edge - https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/17440391/

**Comment by annevk** [comment]
@shacharz great, thanks! Could you file a bug against https://bugs.webkit.org/enter_bug.cgi?product=WebKit&component=HTML%20DOM too?

**Comment by shacharz** [comment]
> @shacharz great, thanks! Could you file a bug against https://bugs.webkit.org/enter_bug.cgi?product=WebKit&component=HTML%20DOM too?

updated

**Comment by annevk** [comment]
Thanks @benjamingr and @shacharz; hopefully this small change is picked up quickly by everyone.

**Pull request merged**